src/js/login.js

```diff
@@ -1,6 +1,7 @@
 import _ from "lodash";
 import $ from "jquery";
 import avatarWindow from "./avatar";
+const url_config = require(".././../url_config.json");
```

Comment:
Please use camel case and import statement (to make import statement work, use js file instead of json).